apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/(marketplace)/components/list-button.tsx

```diff
@@ -40,14 +46,35 @@ export const CreateListingButton: React.FC<CreateListingButtonProps> = ({
         </SheetTrigger>
         <SheetContent className="w-full overflow-y-auto sm:min-w-[540px] lg:min-w-[700px]">
           <SheetHeader>
-            <SheetTitle className="mb-5 text-left">{createText}</SheetTitle>
+            <SheetTitle className="text-left">{createText}</SheetTitle>
           </SheetHeader>
-          <CreateListingsForm
-            contract={contract}
-            type={type}
-            actionText={createText}
-            setOpen={setOpen}
+          <TabButtons
+            tabs={LISTING_MODES.map((mode) => ({
+              name: mode,
+              isActive: mode === listingMode,
+              onClick: () => setListingMode(mode),
+              isEnabled: true,
+            }))}
+            tabClassName="text-sm gap-2 !text-sm"
+            tabContainerClassName="px-3 pt-1.5 gap-0.5"
           />
+          <div className="mt-5 px-3">
```

Comment:
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/yNOf1svJ8o3zjO7zQouZ/538a492e-d859-4c90-9a27-1f76e79300b0.png)

remove `px-3`

Reply:
Wanted to make it looks like a child of the Tab's container but okay

Reply:
update